codice/ddf
# DDF-5557 add explicit rendering to cesium viewer

#### What does this PR do?
Enables the `requestRenderMode` on the Cesium Viewer, which enables explicit rendering on the 3-D map, improving performance and reducing CPU usage. 
See [Improving Performance with Explicit Rendering](https://cesium.com/blog/2018/01/24/cesium-scene-rendering-performance/)

#### Who is reviewing it? 

@andrewzimmer 
@willwill96 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@bdthomson


#### How should this be tested?

* Launch DDF Intrigue in the web browser
* Open Activity Monitor (or Task manager, or any other CPU monitoring tool)
* Notice the reduced CPU load of the browser
**Note**: You may want to run a benchmarking test prior to testing this change, to see what the CPU usage is like on your machine without the change

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #5557 

#### Screenshots
**Before**
Brave Browser Helper (Renderer) - 57.5% CPU
Brave Browser Helper (GPU) - 20.8 % CPU 
![image](https://user-images.githubusercontent.com/25390369/68323606-030b2800-0083-11ea-8efd-3e6b97051ce6.png)


**After**
Brave Browser Helper (Renderer) - 12.6% CPU
Brave Browser Helper (GPU) - 1.1 % CPU 
![image](https://user-images.githubusercontent.com/25390369/68323685-2f26a900-0083-11ea-9fa1-9b1ca91074f9.png)



#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
